### DISCUSSION TITLE
Is it compatible with more low-version browsers

### DISCUSSION THREAD

**Discussion by DreamPWJ** [arthelokyo/astrowind#183] [Q&A]
There are many style problems displayed in browsers with lower versions, or the white screen is not displayed, etc.

**Comment by vasfvitor** [comment]
Could you share specifically what browsers versions you tested and the problems found?

**Comment by DreamPWJ** [comment]
> Could you share specifically what browsers versions you tested and the problems found?

WeChat  and  DingTalk browser

**Comment by DreamPWJ** [comment]
support vendor-prefix？

**Comment by prototypa** [comment]
Hi @DreamPWJ 

Thanks for bringing this issue up.

Much of the theme's browser compatibility depends on [Tailwind CSS's browser support](https://tailwindcss.com/docs/browser-support). But also many of the more advanced Tailwind CSS features could be replaced by more compatible ones in some cases.

I will review in detail and let you know.

**Comment by DreamPWJ** [comment]
When I use the PC WeChat browser to open it, the screen is white and nothing is displayed. When I open it with a DingTalk  on the phone, there are styles that do not work, such as menu buttons. These two applications are the largest in China, so they are often used ，They have a customized browser kernel @prototypa

**Comment by jjaimealeman** [comment]
As mentioned, TailwindCSS compatibility is based on modern browsers.

https://tailwindcss.com/docs/browser-support
